src/watcher.js

```diff
@@ -1,22 +1,29 @@
-const debug = require('./utils/debug');
-const SafeEventEmitter = require('./utils/safe-event-emitter');
+import debug from './utils/debug.js';
+import SafeEventEmitter from './utils/safe-event-emitter.js';
+
+import channelWatcher from './watchers/channel.js';
+import clipsWatcher from './watchers/clips.js';
+import chatWatcher from './watchers/chat.js';
+import conversationWatcher from './watchers/conversations.js';
+import routesWatcher from './watchers/routes.js';
+
 
 const CLIPS_HOSTNAME = 'clips.twitch.tv';
 
 class Watcher extends SafeEventEmitter {
     setup() {
-        require('./watchers/channel')(this);
+        channelWatcher(this);
 
         if (window.location.hostname === CLIPS_HOSTNAME) {
-            require('./watchers/clips')(this);
+            clipsWatcher(this);
```

Comment:
we intentionally lazy load to prevent loading code inapplicable to the page. let's revert this back to dynamic imports

Reply:
should be using dynamic imports now